src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServerWrapper.cs

```diff
@@ -35,19 +33,16 @@ private RazorLanguageServerWrapper(RazorLanguageServer innerServer)
     public static RazorLanguageServerWrapper Create(
         Stream input,
         Stream output,
-        Trace trace,
+        LoggerWrapper razorLogger,
```

Comment:
Is it possible to choose one interface or the other rather than the implementation bridge, `LoggerWrapper`?

Reply:
My desire is to unify on ILspLogger eventually, but I didn't want to bloat the already huge changeset for CLaSP with that change, so I figured we could [do that as a follow-up item](https://github.com/dotnet/razor-tooling/issues/6894).